webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/snippet/administration/TechniqueLibraryManagement.scala

```diff
@@ -532,7 +533,7 @@ class TechniqueLibraryManagement extends DispatchSnippet with Loggable {
                     .attempt("highlight", {}, 2000)}, 100)"""
                 refreshTree() & JsRaw(
                   jsString
-                    .format(sourceactiveTechniqueId)
+                    .format(sourceactiveTechniqueId) // JsRaw ok, comes from json
```

Comment:
If I understand correctly, the `sourceactiveTechniqueId` is a value in the "json object" that is passed when moving an active technique. I think it would still be unsafe if not escaped e.g. when the json is `{"sourceactiveTechniqueId":"]\"alert(1);})//"}`